.test-env

```diff
@@ -1,6 +1,6 @@
 # Configs for testing repo download:
 SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
-SDK_TESTING_BRANCH="master"
+SDK_TESTING_BRANCH="simulate-endpoint-tests"
```

Comment:
remember to change this back

Reply:
i looked at the trap ray

Reply:
https://github.com/algorand/py-algorand-sdk/pull/445